apps/dcos_l4lb/src/dcos_l4lb_lashup_vip_listener.erl

```diff
@@ -427,7 +425,10 @@ test_ip6(IP6, _, _) ->
     list_to_tuple(NextIP6).
 
 process_vips_tcp_test() ->
-    process_vips(tcp).
+    State = process_vips(tcp),
```

Comment:
would it make sense to also assertNotEqual([], ets:tab2list(name_to_ip)) here?

Reply:
There is an assert for that in `process_vip/1`.

https://github.com/dcos/dcos-net/pull/69/files#diff-599c9ad6aeffb3325a8c2108bcc67ed8R459